src/ui/public/utils/lodash-mixins/collection.js

```diff
@@ -1,4 +1,4 @@
-export default function (_) {
```

Comment:
All of these functions are named `extend`.  I think you meant to name them the same as the file name?